<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1031</width>
    <height>773</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyWin</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>221</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>설정</string>
    </property>
    <widget class="QPushButton" name="btnMK">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>매경</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnStart">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>거래시작</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnStop">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>거래중지</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnReStart">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>재시작</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSysTrading">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>201</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>시스템 트레이딩</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>221</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>계좌정보</string>
    </property>
    <widget class="QLabel" name="label2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>64</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>계좌번호</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbAcctNo">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>64</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>D2예수금</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leD2Price">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>64</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>투입금</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lePrice">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>451</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>거래</string>
    </property>
    <widget class="QLabel" name="label4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>조건검색명</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbCdtNm">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>341</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>64</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>총투입금</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leTotalPrice">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label6">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>50</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>수익금</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leProfits">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>50</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label7">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>50</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>수익률</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leProfitsPc">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>50</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>451</width>
      <height>451</height>
     </rect>
    </property>
    <property name="title">
     <string>거래정보</string>
    </property>
    <widget class="QLabel" name="label8">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>64</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>매수종목</string>
     </property>
    </widget>
    <widget class="QLabel" name="label9">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>30</y>
       <width>64</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>매도종목</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="pteBuyLog">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>211</width>
       <height>391</height>
      </rect>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="pteSellLog">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>50</y>
       <width>211</width>
       <height>391</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>541</width>
      <height>711</height>
     </rect>
    </property>
    <property name="title">
     <string>내역</string>
    </property>
    <widget class="QPlainTextEdit" name="pteLog">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>521</width>
       <height>681</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1031</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
